doc/user/content/sql/create-cluster-replica.md

```diff
@@ -47,6 +47,11 @@ _replica_name_ | A name for this replica.
 
 {{% replica-options %}}
 
+{{< note >}}
+If you do not specify an availability zone,  Materialize should automatically assign the availability zone with the least existing replicas. To view all replica availability zones, use
```

Comment:
```suggestion
If you do not specify an availability zone,  Materialize will automatically assign the availability zone with the least existing replicas. To view all replica availability zones, use
```
We have no reason to imply that there is uncertainty around this, right?

Also, maybe we can make it clearer that the AZ auto assignment only considers replicas in the current cluster, rather than all replicas? If you create two clusters with one replica each, it is entirely possible that both will be assigned to the same AZ.

Reply:
That is the language in `/developer/design/20220413_cluster_replica.md` but I agree it sounds ambiguous and can change that.

To clarify - AZ auto assignment only applies to an individual `CREATE_CLUSTER` instance?

Reply:
> To clarify - AZ auto assignment only applies to an individual CREATE_CLUSTER instance?

If by that you mean it only takes into account replicas created in the same `CREATE CLUSTER` statement, then no, it takes into account all replicas existing in the cluster.

Suppose you have three AZs and run the following statements:
```
CREATE CLUSTER c1 REPLICAS (r1 (SIZE 'small'));
CREATE CLUSTER REPLICA c1.r2 SIZE 'small';
CREATE CLUSTER REPLICA c1.r3 SIZE 'small';
```

... then each replica will end up in a different AZ, because they all belong to the same cluster `c2`.

But if you run these statements:
```
CREATE CLUSTER c1 REPLICAS (r1 (SIZE 'small'));
CREATE CLUSTER c2 REPLICAS (r2 (SIZE 'small'));
CREATE CLUSTER c3 REPLICAS (r3 (SIZE 'small'));
```

... then all three could end up in the same AZ because for each of them Materialize sees that it is the first replica in the cluster, so it is unconstrained in which AZs it can pick.

Hope that clears it up!